ci/build_libtelio.py

```diff
@@ -317,7 +329,71 @@ def main() -> None:
         assert False, f"command '{args.command}' not supported"
 
 
+def try_download_artifacts(
+    tag_prefix, path_to_save, target_arch, target_os, moose=False
+):
+    def get_token():
+        if "LLH_GROUP_TOKEN_FLAKY_TESTS" in os.environ:
+            return os.environ["LLH_GROUP_TOKEN_FLAKY_TESTS"]
+        return getpass.getpass("Enter Gitlab API access token:")
+
+    token = get_token()
+
+    if target_os == "uniffi" and moose:
+        raise ValueError(
+            "Cannot download artifacts for uniffi and moose at the same time"
+        )
+
+    try:
+        if ArtifactsDownloader(
+            target_os,
+            target_arch,
+            token,
+            path_to_save,
+            tag_prefix,
+        ).download():
+            if moose and target_os in ["linux", "windows", "android"]:
+                moose_utils.fetch_moose_dependencies(target_os, MOOSE_MAP[target_arch])
+            return True
+        else:
+            print(f'Failed to download artifacts from "{tag_prefix}" pipeline')
+    except Exception as e:
+        print(f'Error while downloading artifacts from "{tag_prefix}" pipeline: {e}')
+
+    return False
+
+
+def exec_bindings(args):
+    if args.try_fetch_from_pipeline:
+        if try_download_artifacts(
+            args.try_fetch_from_pipeline,
+            PROJECT_ROOT,
+            target_arch=None,
+            target_os="uniffi",
+        ):
+            return
+
+    print("Proceeding with generating uniffi ...")
```

Comment:
Same as https://github.com/NordSecurity/libtelio/pull/543/files#r1622533016.

Reply:
Same as https://github.com/NordSecurity/libtelio/pull/543#discussion_r1616764373

Reply:
oh sry didn't see it.